modules/admin-ui/src/main/webapp/scripts/shared/directives/oldDropdownDirective.js

```diff
@@ -28,9 +28,12 @@ angular.module('adminNg.directives')
     },
     link: function ($scope, element) {
       element.on('click', function (event) {
+        var is_active = angular.element(this).hasClass('active');
         angular.element('[old-admin-ng-dropdown]').removeClass('active');
```

Comment:
Instead of re-activating, it might be simpler to spare `this` from deactivation instead?
```suggestion
        angular.element('[old-admin-ng-dropdown]').not(this).removeClass('active');
```